### ISSUE TITLE
aria-proptypes support for idlist

### ISSUE THREAD

**Issue by oliviertassinari** [evcohen/eslint-plugin-jsx-a11y#454]
I have noticed the following errors on Material-UI side trying to upgrade this dependency:
![capture d ecran 2018-07-03 a 21 53 16](https://user-images.githubusercontent.com/3165635/42241634-8c348bf8-7f0b-11e8-886c-6bc9a84f1ee8.png)

The following aria seem to not be supported as expected:
- `aria-labelledby`
- `aria-controls`
- `aria-describedby`

[arial-query returns](https://github.com/A11yance/aria-query/blob/7d8b314b4a0e90d47fd54fc4c7fa7400c8b3a835/src/ariaPropsMap.js#L115) the type `idlist`. Something that is not supported in the implementation: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/b800f40a2a69ad48015ae9226fbe879f946757ed/src/rules/aria-proptypes.js#L32

**Comment by ljharb** [comment]
This is affecting the airbnb config as well, fwiw.

**Comment by jessebeach** [comment]
I'll get this fixed tonight.

6.2.0 update is on the way :)

**Comment by jessebeach** [comment]
Here's the change in aria-query that is causing the errors: https://github.com/A11yance/aria-query/commit/6978637765f455e5be36dbbed1e4ff9ca9957f8b

What I don't yet understand is why none of the tests in this repo caught this. I don't want to simply update aria-query without getting a test case to fail here first so we don't have something like this happen again in the future.

**Comment by ljharb** [comment]
Closed in #456.
